docs/jupyterlab-spack.md

```diff
@@ -0,0 +1,39 @@
+# JupyterLab (spack)
+
+The JupyterLab app that runs via spack has some options in addition to the basic
+CPU and time allocations. These allow you to launch a session with an
+alternative configuration that you define. 
+
+If you just want to use the default packages included in the app, including
+`numpy`, `pandas`, `matplotlib`, and `scipy`, then you don't need to worry about
+these settings. If you want to define your own environment with spack, or you
+have been instructed that an alternative environment has been prepared for your
+class, read on.
+
+## Alternative Spack Installation
+
+The package manager that we use in HUIT Open OnDemand,
+[Spack](https://spack.io/), has a self-contained installation in the file
+system. That means that within your own home directory, or within a course
+shared folder, you can set up a different Spack installation with packages and
+environments that you define, and which can draw on the software already
+installed globally. The JupyterLab (spack) setup allows you to specify an
+alternative Spack path to use, so that the same app configuration can be used
+with these other installations.
+
+When setting this option, you must already have an alternative Spack
+installation set up, and the installation must include a named environment that
+includes the `py-jupyterlab` package. You can refer to this installation by its
+absolute path, beginning with `/shared`
+
+## Alternative Spack Environment
```

Comment:
I think we should reference the `package-management.md` doc here since it outlines some spack environment best practices and caveats.